android/src/main/java/com/chargebee/android/utils/Converter.kt

```diff
@@ -0,0 +1,15 @@
+package com.chargebee.android.utils
+
+import com.facebook.react.bridge.WritableArray
+import com.facebook.react.bridge.WritableNativeArray
+
+internal fun convertArrayToWritableArray(array: Array<Any?>): WritableArray {
```

Comment:
If we're only looking for `String` (or `null`), should `array` be of type `Array<String>`? 🤔

Reply:
Let me try this out.

Reply:
This is done.